.gitignore

```diff
@@ -23,6 +23,8 @@ package-lock.json
 /build/index.html
 /build*
 /build/*
+.env
+/src/config/constants.jsx
```

Comment:
why constants.jsx defined in .gitignore file and what it stores?

Reply:
Ops, my mistake. I stored here URLs as a constants. I will fix it.